devops/providers/azure-devops/templates/infrastructure/scripts/test-integration.yml

```diff
@@ -0,0 +1,38 @@
+parameters:
+  skip: false
+
+steps:
+  - task: AzureCLI@1
+    displayName: 'Integration Test Terraform Template'
+    condition: not(coalesce(variables.SKIP_TESTS, ${{ parameters.skip }}))
```

Comment:
Quick opportunity to fix this issue here: https://github.com/microsoft/cobalt/issues/309

I *think* it would look like the following:
`condition: and(success(), not(coalesce(variables.SKIP_TESTS, ${{ parameters.skip }})))`

Reply:
Agreed, but it's technically out of scope for this change. Might be worth looking at the conditionality a bit more to see if it makes sense at this point or maybe could be handled before even entering the template. Would also help to isolate that testing and exercise all of the paths more.